libs/langchain/langchain/embeddings/bookend.py

```diff
@@ -0,0 +1,84 @@
+"""Wrapper around Bookend AI embedding models."""
+
+import json
+import requests
+
+from langchain.pydantic_v1 import BaseModel, Field
+from langchain.schema.embeddings import Embeddings
+from typing import Any, List
+
+API_URL = "https://api.bookend.ai/"
+DEFAULT_TASK = "embeddings"
+PATH = "/models/predict"
+
+
+class BookendEmbeddings(BaseModel, Embeddings):
+    """Bookend AI sentence_transformers embedding models.
+
+    Example:
+        .. code-block:: python
```

Comment:
need new line between code-block and "from langchain.em..." (otherwise won't render as code block in api docs)